static/app/utils/replays/hooks/useActiveTabFromLocation.tsx

```diff
@@ -4,7 +4,7 @@ const DEFAULT_TAB = ReplayTabs.CONSOLE;
 
 function useActiveTabFromLocation() {
   const hash = location.hash.replace(/^#/, '');
-  const tabFromHash = isReplayTab(hash) ? hash : DEFAULT_TAB;
+  const tabFromHash = isReplayTab(hash) ? hash.replace('%20', ' ') : DEFAULT_TAB;
```

Comment:
We should use `decodeURIComponent` -- I would call it on the line above so that `hash` is decoded.

Reply:
I'll merge #36240 after this PR, so we'll have it fixed up at that time.